apps/api/src/app/organization/organization.controller.ts

```diff
@@ -154,4 +157,14 @@ export class OrganizationController {
       })
     );
   }
+
+  @Get('/in-app/activated')
```

Comment:
I am still not sure under what controller to add validation of the in-app initialization. Because neither the organization nor the subscriber feels entirely correct. it is kinda related to both of them and neither of them.
will love to hear new perspectives.

Reply:
Moved it to integration controller